chapters/methodology/theory.tex

```diff
@@ -102,21 +121,24 @@ \subsection{The $\Omega$ Methods}
 the $\Omega$-methods will use a version of the adjoint scalar flux to
 consistently bias a Monte Carlo problem with the intention of reducing the
 variance. In section \ref{sec:Importance} the concept of importance was
-introduced. Notably, it was shown that the adjoint flux is a good marker for
+introduced. Notably, it was shown that the adjoint flux is a good marker 
 for the likelihood of particles to contribute to a tally, which is the
 particle's importance. It was also shown that the
 product of the forward and adjoint fluxes generates a pseudo-particle flux called
 the contributon flux, where contributons are ``importance particles''.
 These importance particles can be used to show preferential flow paths from a
 source to a tally or desired location.
+% Yes, this is the kind of material for this section. Good description!
 
 By using a version of the adjoint scalar flux that has been formulated with the
 contributon flux, the direction of particle flow will be incorporated into the
-importance map, and consequently also the
-variance reduction parameters. By using a variant of the adjoint scalar flux,
-the method will still maintain the importance relationship between particles and
+importance map and, consequently, the
+variance reduction parameters. By using this variant of the adjoint scalar flux,
+the method will maintain the importance relationship between particles and
 their proximity to the adjoint source location, or the forward response
-location. The adjusted adjoint scalar flux quantity, or the $\Omega$-adjoint
+location. 
+% I'm a little confused by the above sentence...
```

Comment:
"By using this variant of the adjoint scalar flux,
the method will maintain the importance relationship between particles and
their proximity to the adjoint source location, or the forward response
location. "
I'm not sure what 
(1) maintaining the importance relationship means. maintaining compared to what? the forward flux? the regular adjoint flux? some other thing?
(2) is it maintaining both the relationship between the adjoint source and the forward response, one, the other? how do you know which? 

I just didn't really understand what the sentence means or what it's trying to convey